### PR TITLE
Ruby bindings added

### PR DESCRIPTION
**Pull request by sashs**
Ruby bindings added

### CODE REVIEW

File: bindings/ruby/unicorn_gem/lib/unicorn/version.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module Unicorn
+  VERSION = "0.9.0"
```
Comment (aquynh) [inline bindings/ruby/unicorn_gem/lib/unicorn/version.rb]: the core was bumped up to 1.0 long ago, can you change this?

File: bindings/ruby/unicorn_gem/ext/unicorn.c [hunk 2]
```diff
@@ -0,0 +1,454 @@
...
+    rb_define_method(UcClass, "mem_read", m_uc_mem_read, 2);
+    rb_define_method(UcClass, "mem_write", m_uc_mem_write, 2);
+    rb_define_method(UcClass, "mem_map", m_uc_mem_map, -1);
+    //rb_define_method(UcClass, "mem_map_ptr", m_uc_mem_map_pzt, 4);
+    rb_define_method(UcClass, "mem_unmap", m_uc_mem_unmap, 2);
+    rb_define_method(UcClass, "mem_protect", m_uc_mem_protect, 3);
+    rb_define_method(UcClass, "hook_add", m_uc_hook_add, -1);
+    rb_define_method(UcClass, "hook_del", m_uc_hook_del, 1);
+    rb_define_method(UcClass, "query", m_uc_hook_del, 1);
+
```
Comment (aquynh) [inline bindings/ruby/unicorn_gem/ext/unicorn.c]: can you  remove these blank lines?

**Comment by sashs** [comment]
Done

**Comment by aquynh** [comment]
wow, this is a lot of hardwork! merged, thanks!

also updated CREDITS.TXT with your name credited: https://github.com/unicorn-engine/unicorn/commit/c49c7f7c075dec3003c7a0782754a3d8cf04b611.

**Comment by sashs** [comment]
Thanks!
I a few days I will commit sample files for the other architectuers.

**Pull request merged**